package.json

```diff
@@ -10,6 +10,7 @@
     "dev": "gulp --dev && gulp watch",
     "gulp-build": "gulp -b --dev",
     "gulp-watch": "gulp watch",
+    "lint": "eslint **/*.js",
     "test": "echo \"Error: no test specified\" && exit 1"
   },
```

Comment:
Hi @tywrenn  and @bradymiller 

Finally, I add the `npm run lint` to lint all javascript files, please take a look.

Reply:
neat, then we can run this from travis when this is in the codebase

Reply:
Yeah, should I follow the .travis.yml file to add my command?